modules/eventing/pages/eventing-handler-ConvertBucketToCollections.adoc

```diff
@@ -23,6 +23,8 @@
 ** 250M small documents: takes 44 minutes to reorganize a bucket with 80 types into a new bucket with 80 collections.
 ** 1B small documents: takes 3 hours to reorganize a bucket with 80 types into a new bucket with 80 collections.
 
+NOTE: If you alter this function and attempt to run this Eventing function in a single bucket you will have to disable recursion checks. Refer to xref:eventing:troubleshooting-best-practices.adoc[Disabling infinite recursion checks]. In this case always test your Eventing Function on a non-production system to ensure you do not mistakenly create an infinite recursion loop.
```

Comment:
```suggestion
NOTE: If you alter this function and attempt to run this Eventing function in a single bucket you will have to disable recursion checks. 
Refer to xref:eventing:troubleshooting-best-practices.adoc[Disabling infinite recursion checks]. 
In this case, always test your Eventing Function on a non-production system to ensure you do not mistakenly create an infinite recursion loop.
```

Reply:
Try to keep the text to one sentence per line. 
It makes it easier for the Github diff engine to separate out the changes.